### DISCUSSION TITLE
Error during Update - Now can't Login !!!!

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#750] [Q&A (Community Support)]
Tried updating to latest release get a 'Red screen of death' during update. Now when I tried to log back in get same screen - See atached screen shot.
![Table-dbs12288402-pnptc_login_lockouts-doesn-t-exist-500-Internal-Server-Error-](https://github.com/user-attachments/assets/c968be4c-fa8f-4e8b-905a-fbe8f07c7977)

**Comment by nikosdion** [comment]
How about download 1.2.1, extract it over your installation, and try again.

**Comment by david-unwin** [comment]
I assume all my previous settings will still be there after the extraction and upload?
Kind regards

**Reply by nikosdion** [reply]
Correct.

**Comment by david-unwin** [comment]
I tried uploading the latest version overwriting all the files. But I still get the same screen shot as above. So I then deleted all the files and re-uploaded. I then get this  message This page isn’t working admin.cirenopenair.org.uk is currently unable to handle this request.
HTTP ERROR 500  . 
Any ideas?

**Reply by nikosdion** [reply]
You can't get the same message after uploading 1.2.1.

If you do, PHP is not seeing the new files. Ask your host how to reset PHP's OPcache.
